src/ManagerStandalone.sol

```diff
@@ -216,13 +224,14 @@ contract ManagerStandalone is IManagerStandalone, Manager, Implementation {
     function _checkThresholdInvariants() internal view {
         _Threshold storage _threshold = _getThresholdStorage();
         address[] storage _enabledEndpoints = _getEnabledEndpointsStorage();
+        address[] storage _registeredEndpoints = _getRegisteredEndpointsStorage();
 
         // invariant: threshold <= enabledEndpoints.length
         if (_threshold.num > _enabledEndpoints.length) {
             revert ThresholdTooHigh(_threshold.num, _enabledEndpoints.length);
         }
 
-        if (_enabledEndpoints.length > 0) {
+        if (_registeredEndpoints.length > 0) {
```

Comment:
Yeah, I think this makes sense to change. Before, I couldn't perform a migrate because this let's us migrate when none of the endpoints are turned on.